ln-gateway/src/lib.rs

```diff
@@ -85,7 +85,11 @@ macro_rules! impl_gateway_request_trait {
         }
     };
 }
-impl_gateway_request_trait!(HtlcAccepted, Preimage, GatewayRequest::HtlcAccepted);
+impl_gateway_request_trait!(
+    HtlcAccepted,
+    outgoing::Preimage,
```

Comment:
@justinmoon, I'm proposing to send [`outgoing::Preimage`](https://github.com/fedimint/fedimint/blob/884206fb9d43542559c28c6ee3d143a6a50d9fd8/modules/fedimint-ln/src/contracts/outgoing.rs#L28) as response over the gateway server `HtlcAccepted` channel. We previously would send [`incoming::Preimage`](https://github.com/fedimint/fedimint/blob/master/modules/fedimint-ln/src/contracts/incoming.rs#L96).

Do you know if we have any dependencies on this observable response type?

Reply:
Update: I'm learning the hard way how intricate these systems are connected. By responding with a serialized pubkey as preimage, I broke CLN rpc plugin given expectations for a pubkey preimage.

I've since reverted the messaging change, defering any preimage related discussions to #605